modules/lunamediahbBidAdapter.md

```diff
@@ -0,0 +1,80 @@
+# Overview
+
+```
+Module Name: lunamedia Bidder Adapter
+Module Type: lunamedia Bidder Adapter
+Maintainer: support@lunamedia.io
+```
+
+# Description
+
+Module that connects to lunamedia demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+                // Will return static test banner
+                {
+                    code: 'adunit1',
+                    mediaTypes: {
+                        banner: {
+                            sizes: [[300, 250]],
+                        }
+                    },
+                    bids: [
+                        {
+                            bidder: 'lunamediahb',
+                            params: {
+                                placementId: 0,
+                                traffic: 'banner'
```

Comment:
Is there a reason for having the publisher pass 'traffic' as a param value, wouldn't this be derived from the mediaTypes in the adUnit and passed to your adapter in the bidRequest?

Reply:
It's easier to use traffic parameter then check every type for existence in mediatype

Reply:
The mediaType is passed to your adapter in the bid request:

`
const mediaTypes = req.mediaTypes;
if (mediaTypes.native) {

} else if (mediaTypes.banner) {

} else if (mediaTypes.video) {

}
`

Could you update to check for this instead of using a traffic param?

Reply:
Done. And about lunamedia adapter, they are both related to lunamedia, but current adapter is made to support brand new solution, previous one is kept working to